### PR TITLE
Custom default language specified by TLDR_LANGUAGE

### PR DESCRIPTION
**Pull request by nebnola**
Implements the suggestion from issue #156.

I hope I understood your comments correctly. In addition to using `TLDR_LANGUAGE`, I also fixed what I think is an error:
https://github.com/tldr-pages/tldr-python-client/blob/f24f81fb01d2b1bb69840bc6f98921fcbd700033/tldr.py#L195-L198
This meant that if `default_lang` was in `languages`, then `languages` would get set to `[]`, ignoring both variables.
If I misunderstood something and this was intended, then feel free of course to change that part back

### CODE REVIEW

File: tests/test_tldr.py [hunk 1]
```diff
@@ -61,6 +61,22 @@ def test_get_default_language_unset(monkeypatch):
...
+    for name, var in [("TLDR_LANGUAGE", tldr_language),
+                      ("LANGUAGE", language),
+                      ("LANG", lang)]:
+        # Unset environment variable if their value is given as None
+        if var is None:
+            monkeypatch.delenv(name)
+        else:
+            monkeypatch.setenv(name, var)
+    assert tldr.get_language_list() == expected
+
```
Comment (bl-ue) [inline tests/test_tldr.py:79]: The CI is failing because there ought to be two blank lines between each function:
```suggestion


```

**Comment by nebnola** [comment]
Do you guys understand where this error comes from? I don't see what it has to do with the changes, but are there any further changes I should make?

**Comment by bl-ue** [comment]
Well, Python 3.5 [is EOL](https://docs.python.org/3.5/) (End-of-Life), and I think we should remove support for it.  You can open an additional PR to remove 3.5 from [here](https://github.com/tldr-pages/tldr-python-client/blob/8744b1b5136f4a812c3506361c31cf2a89d151e5/.github/workflows/test.yml#L11), and after we merge that into `main`, you can rebase `main` into this PR and the CI should be all green.

File: tldr.py [hunk 2]
```diff
@@ -338,8 +346,12 @@ def output(page):
...
 def update_cache(language=None):
     if language is None:
-        default_lang = get_default_language()
-        language = default_lang if default_lang is not None else 'en'
+        tldr_language = os.environ.get("TLDR_LANGUAGE")
+        if tldr_language is not None:
+            language = tldr_language
+        else:
+            default_lang = get_default_language()
+            language = default_lang if default_lang is not None else 'en'
```
Comment (MasterOdin) [inline tldr.py:354]: I believe we can shorten this to be just two lines?
```suggestion
        tldr_language = os.environ.get("TLDR_LANGUAGE", get_default_language())
        language = default_lang if default_lang is not None else 'en'
```

**Review by MasterOdin — CHANGES_REQUESTED** [review]

File: tldr.py (hunk 2 above)
Comment (bl-ue) [reply tldr.py:354]: You could shorten a tad bit further by saying `default_lang if default_lang else 'en'` (i.e. removing ` is not None`).

Comment (nebnola) [reply tldr.py]: I added this suggestion, except of course that it should be `tldr_language` instead of `default_lang` in the last line

**Review by MasterOdin — APPROVED** [review]
